.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "editor.tabSize": 2,
+    "editor.detectIndentation": false
+}
```

Comment:
You can add `"files.insertFinalNewline": true,` to auto insert a newline at end of file.
And you can add this in **cmd + shift + p -> open settings**